data/rulesets.js

```diff
@@ -292,6 +292,10 @@ let BattleFormats = {
 			if (!template.nfe || futureGenEvo) {
 				return [set.species + " doesn't have an evolution family."];
 			}
+			// Temporary hack for LC past-gen formats and other mashups
+			if (set.level !== 5) {
+				return [`${set.species} needs to be level 5.`];
```

Comment:
LC requires level 5, it's not just a cap?

Reply:
True, fixed